documentation/staging/content/userguide/managing-domains/model-in-image/overview.md

```diff
@@ -24,36 +24,49 @@ Unlike Domain in PV and Domain in Image, Model in Image eliminates the need to p
 It enables:
 
  - Defining a WebLogic domain home configuration using WebLogic Deploy Tool (WDT) model files and application archives.
- - Embedding model files and archives in a custom container image, and using the WebLogic Image Tool (WIT) to generate this image.
- - Supplying additional model files using a Kubernetes ConfigMap.
- - Supplying Kubernetes Secrets that resolve macro references within the models. For example, a secret can be used to supply a database credential.
- - Updating WDT model files at runtime. For example, you can add a data source to a running domain. See [Runtime updates](#runtime-updates) for details.
+ - Embedding these files in a single image that also contains a WebLogic installation,
+   and using the WebLogic Image Tool (WIT) to generate this image. Or, alternatively,
+   embedding the files in one or more small images.
```

Comment:
More "small"... I'm still worried about perspective and would prefer "application-specific" or similar.

Reply:
Fixed.